ch.theologeek.Manuskript.json

```
@@ -0,0 +1,147 @@
+            },
+            {
+              "type": "script",
+              "commands": [
+                "python3 configure.py --sip-module PyQt5.sip -b ${FLATPAK_DEST}/bin -d ${FLATPAK_DEST}/lib/python3.9/site-packages -e ${FLATPAK_DEST}/include -v ${FLATPAK_DEST}/share/sip --stubsdir=${FLATPAK_DEST}/lib/python3.9/site-packages"
+              ],
+              "dest-filename": "configure"
+            }
+          ],
+          "cleanup": [
+            "/bin",
+            "/include"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "enchant",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/AbiWord/enchant/releases/download/v2.3.1/enchant-2.3.1.tar.gz",
+          "sha256": "7b4b1afcf2cd8bfa691deea6188404d337f23174bbc39b9c2add2bf340736e9c"
+        }
+      ]
+    },
+    {
+      "name": "pandoc",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cp -R . /app/"
+      ],
+      "sources": [
+	    {
+          "type": "archive",
+          "url": "https://github.com/jgm/pandoc/releases/download/2.14.2/pandoc-2.14.2-linux-amd64.tar.gz",
+          "sha256": "1a0548b15255b1c11722f3d4e12fc7a652edf8c9a329a8458f1e765517aec1be"
+        }
+      ],
+	  "modules": [
+		{
+		  "name": "texlive-extension",
+		  "buildsystem": "simple",
+		  "build-commands": [
+			"mkdir /app/texlive"
+		  ]
+		}
+	  ]
+    },
+    {
+      "name": "manuskript",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/share/manuskript/",
+        "cp -R . /app/share/manuskript/",
+        "mkdir -p /app/bin/",
+        "ln -s ../share/manuskript/bin/manuskript /app/bin/manuskript"
+      ],
+      "sources": [
+	    {
+          "type": "archive",
+          "url": "https://github.com/olivierkes/manuskript/archive/0.12.0.zip",
+          "sha256": "676da858d0a3bb658234191b2b36c56dbb07f03371469990de34f7701842c441"
+        },
+		{
+		  "type": "file",
+		  "path": "ch.theologeek.Manuskript.desktop"
+		},
+		{
+		  "type": "file",
+		  "path": "ch.theologeek.Manuskript.appdata.xml"
+		}
+      ],
+	  "post-install": [
+		  /* install icons in several resolutions */
+		  "install -Dm644 icons/Manuskript/icon-16px.png /app/share/app-info/icons/flatpak/16x16/ch.theologeek.Manuskript.png",
+		  "install -Dm644 icons/Manuskript/icon-32px.png /app/share/app-info/icons/flatpak/32x32/ch.theologeek.Manuskript.png",
+		  "install -Dm644 icons/Manuskript/icon-64px.png /app/share/app-info/icons/flatpak/64x64/ch.theologeek.Manuskript.png",
+		  "install -Dm644 icons/Manuskript/icon-128px.png /app/share/app-info/icons/flatpak/128x128/ch.theologeek.Manuskript.png",
+		  "install -Dm644 icons/Manuskript/icon-256px.png /app/share/app-info/icons/flatpak/256x256/ch.theologeek.Manuskript.png",
+		  "install -Dm644 icons/Manuskript/icon-512px.png /app/share/app-info/icons/flatpak/512x512/ch.theologeek.Manuskript.png",
```

```suggestion
		  "install -Dm644 icons/Manuskript/icon-512px.png /app/share/icons/hicolor/512x512/apps/ch.theologeek.Manuskript.png",
```

Unless used internally by the app, this should be enough. (Note the different location.)

---

I had this different location used before but the bot failed because he looked up the icons at the other path (currently used).

---

This is because the desktop file is installed at the wrong location. It should be fine once it's fixed.

Files in the `app-info` directory are written by `appstream-compose` executed by flatpak-builder somewhere at the end of the build process.